src/booking.js

```diff
@@ -739,20 +743,28 @@ class RecrasBooking {
         }
     }
 
-    productCounts() {
-        let counts = [];
-        [...this.findElements('[id^="packageline"]')].forEach(line => {
-            counts.push({
-                aantal: (isNaN(parseInt(line.value)) ? 0 : parseInt(line.value)),
-                arrangementsregel_id: parseInt(line.dataset.packageId, 10),
-            });
-        });
-        this.getLinesBookingSize(this.selectedPackage).forEach(line => {
-            counts.push({
+    productCountsBookingSize() {
+        return this.getLinesBookingSize(this.selectedPackage).map(line => (
+            {
                 aantal: this.bookingSize(),
                 arrangementsregel_id: line.id,
-            });
-        });
+            }
+        ));
+    }
+
+    productCountsNoBookingSize() {
```

Comment:
Shouldn't this function have a filter for 'packagelines that don't inherit the booking size'?

Reply:
No, because `this.findElements('[id^="packageline"]')` only concerns elements for packagelines that don't inherit booking size:

![2019-05-21 13 20 56 localhost d03963ecd7a4](https://user-images.githubusercontent.com/3470653/58092061-76030e00-7bcb-11e9-95a2-ccd359a5baa4.png)